src/app/blog/[slug]/page.tsx

```
@@ -0,0 +1,86 @@
+import { getBlogPostBySlug } from '$/services/blog'; // Updated import path
+// BlogPost import removed
+import { notFound } from 'next/navigation';
+// Image import removed
+import type { BlogResponse } from '$/utils/pocketbase/pocketbase-types'; // Corrected path
```

Biome (1.9.4)

[error] This import is unused.

Unused imports might be the result of an incomplete refactoring.
Safe fix: Remove the unused import.


(lint/correctness/noUnusedImports)